csvtable.php

```diff
@@ -0,0 +1,62 @@
+<?php
+$path = $_GET['path'];
```

Comment:
Do not use get or post method, get the file path and id from the database of the current logged in user.

Reply:
There is a list of different users CSV file to update, not only logged-in users